cf_xarray/tests/test_accessor.py

```diff
@@ -665,3 +666,42 @@ def test_standard_name_mapper():
     actual = da.cf.sortby("standard_label")
     expected = da.sortby("label")
     assert_identical(actual, expected)
+
+
+@pytest.mark.parametrize("obj", objects)
+@pytest.mark.parametrize(
+    "attr", ["drop", "drop_vars", "drop_sel", "set_coords", "reset_coords"]
+)
+def test_names_labels(obj, attr):
+
+    # DataArray object has no attribute set_coords
+    if not isinstance(obj, Dataset) and attr == "set_coords":
+        return
+
+    # Get attribute
+    expected = getattr(obj, attr)
+    actual = getattr(obj.cf, attr)
+
+    if attr == "drop_sel":
```

Comment:
I think it would be cleaner if we tested `drop_sel` and `reset_coords` in a different function without parmaeterizing `attr`

``` python
obj.reset_coords(...)
...
obj.drop_sel(Y=50)
```

Reply:
Let's also suppress this warning
```
dropping variables using `drop` will be deprecated; using drop_vars is encouraged.
```

Reply:
OK! I've suppressed both the Future and PendingDeprecation warnings as they are triggered by drop in the tests.